src/components/stepforms/FormulaStepForm.vue

```diff
@@ -65,7 +65,7 @@ export default class FormulaStepForm extends BaseStepForm<FormulaStep> {
     let ret = this.validator({ ...this.editedStep });
     let errors: ErrorObject[] = [];
     try {
-      parse(this.editedStep.formula);
+      parse(this.editedStep.formula.replace(/\[.*?\]/g, 'var'));
```

Comment:
What would you think of adding `[]` to a string constant and use it? It's a genuine question, I have no real strong opinion for this specific case.